admin/services/data-access/pupil-restart.data.service.js

```diff
@@ -115,20 +116,79 @@ pupilRestartDataService.sqlFindLatestRestart = async function (pupilId) {
 
 /**
  * Find pupil's restart codes
- * @param pupilId
  * @return {Promise.<void>}
  */
 pupilRestartDataService.sqlFindRestartCodes = async function () {
   const sql = `
   SELECT 
     id, 
     code, 
-    statusDesc 
+    description
   FROM ${sqlService.adminSchema}.[pupilRestartCode]
-  ORDER BY statusDesc ASC`
+  ORDER BY description ASC`
+  return sqlService.query(sql)
+}
+
+/**
+ * Find pupil's restart reason description by id
+ * @param id
+ * @return {Promise.<void>}
+ */
+pupilRestartDataService.sqlFindRestartReasonDescById = async function (id) {
+  const sql = `
+  SELECT 
+    description
+  FROM ${sqlService.adminSchema}.[pupilRestartReason]
+  WHERE id=@id
+  ORDER BY description ASC`
+  const params = [
+    {
+      name: 'id',
+      value: id,
+      type: TYPES.Int
+    }
+  ]
+  const result = await sqlService.query(sql, params)
+  const obj = R.head(result)
+  return obj && obj['description']
```

Comment:
FYI - `R.prop('description', obj)` would return the property or undefined if the property or the object did not exist.
http://ramdajs.com/docs/#prop

Reply:
@jon-shipley yea i completely forgot about this, it's added